packages/server/lib/automation/cookies.ts

```diff
@@ -5,8 +5,9 @@ import { isHostOnlyCookie } from '../browsers/cdp_automation'
 
 export interface AutomationCookie {
   domain: string
-  expiry: number | null
+  expiry: 'Infinity' | '-Infinity' | number | null
```

Comment:
whats the difference between maxAge & expiry? or is one an alias for the other?

Reply:
They both accomplish the same thing. `expiry` is the full date of when the cookie expires, vs `maxAge` is the time in seconds when the cookie expires. I don't think `expiry` is officially deprecated but since `maxAge` was added most people use that.